.travis.yml

```diff
@@ -23,17 +25,17 @@ script:
 
 after_success:
   - codecov
+  - if [[ $BUILD_INSTALLER == 'yes' ]]; then pip install pynsist; ./script/makeinstaller.sh; fi
 
-before_deploy:
-  - sudo apt-get install nsis
-  - sudo pip install s3cmd
-  - pip install pynsist
-  - "./script/makeinstaller.sh"
+addons:
+  apt:
+    packages:
+    - nsis
 
 deploy:
   provider: script
   skip_cleanup: true
-  script: if [ -n "${TRAVIS_TAG}" ]; then s3cmd put build/nsis/streamlink-*.exe s3://streamlink-builds/release/windows/; else s3cmd put build/nsis/streamlink-*.exe s3://streamlink-builds/nightly/windows/; s3cmd put build/nsis/streamlink-*.exe s3://streamlink-builds/nightly/windows/streamlink-latest.exe; fi
+  script: sudo pip install s3cmd; if [ -n "${TRAVIS_TAG}" ]; then s3cmd put build/nsis/streamlink-*.exe s3://streamlink-builds/release/windows/; else s3cmd put build/nsis/streamlink-*.exe s3://streamlink-builds/nightly/windows/; s3cmd put build/nsis/streamlink-*.exe s3://streamlink-builds/nightly/windows/streamlink-latest.exe; fi
```

Comment:
`local_dir` might work. But that would mean moving the resulting `exe` as the directory it outputs to at the moment gets all of temp files, etc. Would be a lot cleaner though :)